tests/v5/mqtt5_to_mqtt3_adapter_tests.c

```diff
@@ -2686,6 +2686,91 @@ static int s_mqtt5to3_adapter_subscribe_single_success_fn(struct aws_allocator *
 
 AWS_TEST_CASE(mqtt5to3_adapter_subscribe_single_success, s_mqtt5to3_adapter_subscribe_single_success_fn)
 
+/*
+ * This function tests receiving a subscribe acknowlege after disconnecting from
+ * the server.
+ * it expects a AWS_MQTT_QOS_FAILURE return
+ */
+static int s_mqtt5to3_adapter_subscribe_single_success_null_suback_fn(struct aws_allocator *allocator, void *ctx) {
```

Comment:
Probably remove the `success` from the name. It is not testing a success case here.